### DISCUSSION TITLE
Scholar Compass 高校文档

### DISCUSSION THREAD

**Discussion by giscus** [Scholar-Compass/content#1] [General]
# Scholar Compass 高校文档

帮助高三毕业生更好地择校选专业。

https://scholar-compass.github.io/content/

**Comment by YDX-2147483647** [comment]
🤝如果您想改进此项目，请看这里！

- 🔎反馈问题

  滚动到页面最下方（例如这里），可直接评论。

- ✍修改内容

  1. 单击页面右上角的“编辑此页”📝按钮，进入 GitHub 相应源文件。
  2. 单击右上角“Edit this file”🖉按钮，或按<kbd>E</kbd>。
  3. 按照提示 fork、编辑、提交 pull request。

- 🤖对于复杂修改，请参阅[贡献者指南](https://github.com/Scholar-Compass/content/blob/main/CONTRIBUTING.md)。
